### PR TITLE
Shapeshift tx

### LINKED ISSUE #553: Put shapeshift info in tx history

### PR DESCRIPTION
**Pull request by frankiebee**
fixes #553 
screen shot 2016-08-18 at 4 27 03 pm

### CODE REVIEW

**Comment by flyswatter** [comment]
I like the color, I don't like the distance between the + and the number.

**Comment by flyswatter** [comment]
Could you add a state with a shapeshift tx to our `development/states` folder?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Current Master
 
+- Added ShapeShift to the transaction history
+- Added PUB.KEY too ShapeShift request for affiliate program
```
Comment (flyswatter) [inline CHANGELOG.md]: Make this prettier for normal people. `Added affiliate key to shapeshift requests`, maybe?

**Comment by frankiebee** [comment]
@flyswatter yeah ill do that now

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (flyswatter) [inline npm-debug.log]: don't include your debug log. Even maybe add `npm-debug.log` to the `.gitignore` so this doesn't happen again.

**Comment by flyswatter** [comment]
but yeah, that gap between the `+` and the number.. can you reduce that, or is that a side effect of our number formatter?  Maybe need to move the `+` into the number formatter?

**Comment by frankiebee** [comment]
that is a side effect of the number formatter

**Pull request merged**